hubblestack/returners/sqlite.py

```diff
@@ -187,13 +187,17 @@ def get_load(jid, conn=None):
 @_open_close_conn
 def _insert_helper(ret, conn=None):
     log.debug('populating jids table with %s', ret.get('jid'))
+    ret_jid = '' if ret.get('jid') is None else ret.get('jid')
```

Comment:
Dictionary has a simple method, this line can be replaced by:

ret_jid = ret.get('jid', '')

Same for the rest of the lines.